### PR TITLE
Feature/home view cag

### PR DESCRIPTION
**Pull request by cagallo**
## What Features are Being Added?

- Added details to App.js, BookContainer.js, BookCard.js
- Added Router to App.js and BookCard.js

## What is Being Refactored?

- N/A

## Where should the reviewer start?

- App.js, BookContainers.js, BookCard.js

## How should this be manually tested?

- npm start, go to localhost:3000

## What are the relevant tickets?

Create components for Book Club UI

## Any Other Notes/Questions/Problems?

- N/A

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -2,19 +2,106 @@ import '../styles/App.css';
...
+          book_image: 'https://storage.googleapis.com/du-prd/books/images/9780316499774.jpg'
+        }
+      ]
     }
   }
 
   render() {
     return (
-      <h1>book club</h1>
+     <section className='App'>
```
Comment (krdurrett) [inline src/components/App.js:92]: This initial render using switch/routes look perfect!

File: src/components/BookCard.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
 import React from "react";
 import '../styles/BookCard.css';
+import { Link } from 'react-router-dom';
 
-const BookCard = (props) => {
+const BookCard = ({title, author, bookImage, id }) => {
     return (
-        <h3>blah</h3>
+      <section className="book-card">
```
Comment (krdurrett) [inline src/components/BookCard.js:7]: Nice initial layout for book cards

File: src/components/App.js [hunk 3]
```diff
@@ -2,19 +2,106 @@ import '../styles/App.css';
...
+import { Route, Switch } from 'react-router-dom';
 
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-
+      books: [
+        {
```
Comment (Fordo29) [inline src/components/App.js:14]: Great use of dummy data to just get the page set up

File: src/components/BookContainer.js [hunk 4]
```diff
@@ -2,11 +2,22 @@ import React from "react";
 import '../styles/BookContainer.css';
 import BookCard from "./BookCard";
 
-const BookContainer = (props) => {
-
-    return(
-        <h2>Hi Ethan's books container</h2>
+const BookContainer = ({ allBooks }) => {
```
Comment (krdurrett) [inline src/components/BookContainer.js:5]: Nice work mapping through books data to create individual cards

File: src/components/App.js [hunk 5]
```diff
@@ -2,19 +2,106 @@ import '../styles/App.css';
...
     return (
-      <h1>book club</h1>
+     <section className='App'>
+       <Switch>
+        <Route exact path='/'>
+          <BookContainer allBooks={this.state.books}/>
+        </Route>
+        <Route exact path='/favorites' render={() => {
+          return <Favorites />
+        }} />
```
Comment (ectweitmann) [inline src/components/App.js:99]: If these do prove to be equivalent ways of accomplishing the same goal, then at some point we should change one or the other for the sake of cohesion.

File: src/components/App.js [hunk 6]
```diff
@@ -2,19 +2,106 @@ import '../styles/App.css';
...
+        }
+      ]
     }
   }
 
   render() {
     return (
-      <h1>book club</h1>
+     <section className='App'>
+       <Switch>
```
Comment (Fordo29) [inline src/components/App.js:93]: Way to start  with router and switch out of the gate.

**Review by krdurrett — APPROVED** [review]
This initial layout/functionality for the main view of all books looks great so far

**Review by ectweitmann — APPROVED** [review]
This is great Chez, nice work!

File: src/components/BookCard.js [hunk 7]
```diff
@@ -1,9 +1,17 @@
...
+        <img src={bookImage}/>
+        <h2>{title}</h2>
+        <p>{author}</p>
+        <Link to={`/${id}`}>
```
Comment (Fordo29) [inline src/components/BookCard.js:11]: Great learning to see how to link a button

**Pull request merged**